checklists/afd_checklist.en.json

```diff
@@ -197,6 +197,7 @@
             "guid": "49a98f2b-ec22-4a87-9415-6a10b00d6555",
             "id": "A01.30",
             "severity": "High",
+            "graph" : "resources| where type == "microsoft.network/frontdoorwebapplicationfirewallpolicies"| where properties['managedRules']['managedRuleSets'][0]['ruleSetType'] =~ "Microsoft_DefaultRuleSet"| extend compliant = properties['managedRules']['managedRuleSets'][0]['ruleSetType'] =~ "Microsoft_DefaultRuleSet"| distinct id,name,compliant",
```

Comment:
- Please use single quotes (instead of double quotes) inside of the ARG query string
- Pluse use the `=~` operator for the type field